class/defaults.yml

```diff
@@ -1,4 +1,63 @@
 parameters:
   cilium:
     =_metadata: {}
-    namespace: syn-cilium
+    namespace: cilium
```

Comment:
Why not `syn-cilium`?

Reply:
I honestly don't know. I had https://github.com/appuio/component-appuio-cloud/pull/41 in mind. I'm not sure why we did it and if Cilium only works in the `cilium` namespace.

Maybe we shouldn't make the namespace configurable...

Reply:
I made the configuration private.